cypress/integration/step_definitions/backend/ss_delete_usersubscription.cy.js

```diff
@@ -0,0 +1,75 @@
+import { Given, When, Then } from "cypress-cucumber-preprocessor/steps";
+
+let res;
+
+let subscriptionID;
+
+Given("I get the Subscription ID from the userSubscription API", () => {
+  cy.userSubscription().then((response) => {
+    expect(response.status).to.eq(200);
+    res = response
+    subscriptionID = response.body.subscription_id;
+  });
+});
+
+
+When("I send a POST request to {string}", () => {
+
```

Comment:
this method should not work. you re saying {string} but not sending any string as parameter. Is it working?